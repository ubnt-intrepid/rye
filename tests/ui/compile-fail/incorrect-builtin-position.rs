fn main() {}

#[rye::test]
fn builtin_in_closure(cx: &mut rye::Context<'_>) {
    drop(|| {
        skip!();
    });
}

#[rye::test]
fn builtin_in_async_block(cx: &mut rye::Context<'_>) {
    drop(async {
        skip!();
    });
}
