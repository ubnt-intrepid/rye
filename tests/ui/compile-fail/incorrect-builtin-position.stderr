error: builtin macros cannot be used inside of closure or async block
 --> $DIR/incorrect-builtin-position.rs:6:9
  |
6 |         skip!();
  |         ^^^^^^^^

error: builtin macros cannot be used inside of closure or async block
  --> $DIR/incorrect-builtin-position.rs:13:9
   |
13 |         skip!();
   |         ^^^^^^^^
